scripts/lib/CIME/hist_utils.py

```diff
@@ -246,6 +249,9 @@ def _compare_hists(case, from_dir1, from_dir2, suffix1="", suffix2="", outfile_s
         num_compared += len(match_ups)
 
         for hist1, hist2 in match_ups:
+            if not '.nc.' in hist1:
```

Comment:
Wouldn't this fail if hist1 didn't have a suffix, and so ended with just `.nc` rather than `.nc.suffix1`?

I'm wondering if this should be changed to simply `if not '.nc' in hist1:`?

Reply:
fixed, thanks